src/minizinc/CLI/driver.py

```diff
@@ -149,6 +150,18 @@ async def create_process(
             solver (Union[str, Path, None]): Solver configuration string
                 guaranteed by the user to be valid until the process has ended.
         """
+        kwargs = {
+            "stdin": None,
+            "stdout": PIPE,
+            "stderr": PIPE,
+        }
```

Comment:
I don't really understand why we now need to pass in the pipes using a dictionary. I was under the impression that you could use a combination of key-word arguments and a dictionary

Reply:
If we can indeed do that I would prefer to only put variable key word arguments in a dictionary.

Reply:
Yep, you're right. I was trying to reduce code duplication, but having correct type checking is better.